azure-iot-device/tests/iothub/test_sync_clients.py

```diff
@@ -76,11 +76,21 @@ class SharedClientCreateFromConnectionStringTests(object):
     @pytest.mark.parametrize(
         "ca_cert",
         [
-            pytest.param(None, id="No CA certificate"),
-            pytest.param("some-certificate", id="With CA certificate"),
+            pytest.param(None, id=" No CA certificate"),
+            pytest.param("some-certificate", id=" With CA certificate"),
```

Comment:
was the extra space intentional?  Why?

Reply:
Yes, please remove

---
In reply to: [335149871](https://github.com/Azure/azure-iot-sdk-python/pull/331#discussion_r335149871) [](ancestors = 335149871)